api/_mixins/DocumentOrShadowRoot__Document.json

```diff
@@ -463,15 +417,15 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
         }
       },
       "pointerLockElement": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/pointerLockElement",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/pointerLockElement",
```

Comment:
Collector results say this is not supported on IE or Safari iOS, and was introduced in Safari 10.1. https://caniuse.com/pointerlock agrees, so I'd go with that.